sendfile/__init__.py

```diff
@@ -58,9 +56,9 @@ def sendfile(request, filename, attachment=False, attachment_filename=None, mime
     '''
 
     #TODO: PY3Upgrade - Cleanup
-    # Intentially ninja import future libraries into ridiculous __init__.py file
+    # Intentionally ninja import future libraries into ridiculous __init__.py file
     # (python 3 upgrade, py2/3 compatible)
-
+    from builtins import map
```

Comment:
I don't think this function actually calls map anywhere, it's used on line 2 to stitch `VERSION` together into `__version__`

Reply:
Woah. Thanks. Moving too fast.